addons/SimpleTODO/TODOColumn.gd

```diff
@@ -1,27 +1,79 @@
 tool
 extends PanelContainer
 
-onready var name_edit = $VBoxContainer/HBoxContainer2/Name
+onready var head = $VBoxContainer/Head
+onready var top_separator = $VBoxContainer/TopSeparator
+onready var bottom_separator = $VBoxContainer/BottomSeparator
+onready var scroll_container = $VBoxContainer/ScrollContainer
+onready var actions = $VBoxContainer/Actions
+onready var minimize_button = $VBoxContainer/Head/Minimize
+
+onready var name_edit = $VBoxContainer/Head/Name
 onready var item_container = $VBoxContainer/ScrollContainer/Items
-onready var counter = $VBoxContainer/HBoxContainer2/Counter
-onready var delete_button = $VBoxContainer/HBoxContainer/DeleteColumn
+onready var counter = $VBoxContainer/Head/Counter
+onready var delete_button = $VBoxContainer/Actions/DeleteColumn
 onready var timer = $Timer
 
 var undo_redo: UndoRedo
+var minimized = false setget set_minimized
+var main: Control
+
+var item_placement_holder: Panel
+var is_dragging = false
+var initial_item_index = 0
+var current_drag_item_index = 0
+
+var item_margin = 20
+
+func set_minimized(val):
+	minimized = val
+	
+	var new_text = "V"
+	
+	if not val:
+		new_text = "^"
+
+	minimize_button.text = new_text
+	
+	top_separator.visible = not val
+	scroll_container.visible = not val
+	bottom_separator.visible = not val
+	actions.visible = not val
 
 signal delete
 
 func _ready() -> void:
+	item_placement_holder = main.item_placement_holder
 	delete_button.icon = get_icon("Remove", "EditorIcons")
 	counter.rect_min_size.x = delete_button.get_minimum_size().x
+	set_minimized(false)
+
+func _process(delta):
+	if is_dragging:
```

Comment:
You could do `set_process()` to toggle process instead of only using boolean. This way the column wouldn't process needlessly.